shop/src/main/java/ro/msg/learning/shop/models/Location.java

```diff
@@ -0,0 +1,34 @@
+package ro.msg.learning.shop.models;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Data
+@Table(name = "location")
+@Getter @Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class Location extends  BaseEntity{
+
+    private String name;
+    private String addressCountry;
+    private String addressCity;
+    private String addressCounty;
+
+    private String adressStreetAddress;
```

Comment:
You can create a new entity named Address and use here (and on every entity which has these fields) an instance of Address.

Reply:
Created new table `Address` with one-to-one relationship for every table that uses the address